subsys/bluetooth/host/att.c

```diff
@@ -202,7 +202,7 @@ static int process_queue(struct bt_att_chan *chan, struct k_fifo *queue)
 	buf = net_buf_get(queue, K_NO_WAIT);
 	if (buf) {
 		err = chan_send(chan, buf, NULL);
```

Comment:
How about moving this handling into chan_send, where we call l2cap_chan_send? That way we don't have chan_send return different success values depending on the underlying l2cap function call.

Reply:
Done.